documentation/docs/40-best-practices/05-performance.md

```diff
@@ -71,7 +71,7 @@ To run third party scripts in a web worker (which avoids blocking the main threa
 
 ### Selective loading
 
-Code imported with static `import` declarations will be automatically bundled with the rest of your page. If there is a piece of code you need only when some condition is met, use the dynamic `import(...)` form instead.
+Code imported with static `import` declarations will be automatically bundled with the rest of your page. If there is a piece of code you want to lazy load or need only when some condition is met, use the dynamic import form instead: `await import(...)`.
```

Comment:
'If there is a piece of code you want to lazy load or need only when some condition is met' is pretty awkward grammar — the boundaries of the clauses are unclear, and 'lazy load' should either be 'lazy-load' or (better) 'lazily load'

```suggestion
Code imported with static `import` declarations will be automatically bundled with the rest of your page. If there is a piece of code you need only when some condition is met, use the dynamic `import(...)` form to load the component lazily.
```

Reply:
people aren't going to be searching for "lazily" in the docs, so I switched it to "lazy-load"

Reply:
I guess the reason the section title is "selective loading" is that you could be doing it immediately and in that sense it's not lazy. In that sense the suggestion isn't quite accurate, but maybe I'm being overly pedantic about that